test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -5,6 +5,147 @@
  */
 #include "serializer.h"
 
+#include <stack>
+#include <string>
+
+namespace {
+
+const std::string kCounterNamePrefix = "counter_";
+
+/**
+ * Class that controls id creation for counters. Basically, a variable wrapper
+ * that guarantees variable to be incremented.
+ */
+class CounterIdProvider_ {
+public:
+	/** Returns number of id provided. */
+	std::size_t
+	count()
```

Comment:
I suppose there is no need to split return type definition and name of
the class method. We should declare the methods as the following:

```cpp
std::size_t count()
{
	return id;
}
```

Here and below.

Reply:
Fixed.